bip-0349.md

```diff
@@ -0,0 +1,71 @@
+```
+BIP: 349
+Layer: Consensus (soft fork)
+Title: OP_INTERNALKEY
+Author: Brandon Black <freedom@reardencode.com>, Jeremy Rubin <j@rubin.io>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0349
+Status: Draft
+Type: Standards Track
+Created: 2023-12-22
+License: BSD-3-Clause
+```
```

Comment:
I think these changes should make the CI pass:

- use `<pre>` and `</pre>` instead of <code>```</code>
- put each author on a separate line

I also suggest updating the date in the Created header to the day that the proposal was assigned a number as that’s the meaning of the field according to BIP 2.

```suggestion
<pre>
  BIP: 349
  Layer: Consensus (soft fork)
  Title: OP_INTERNALKEY
  Author: Brandon Black <freedom@reardencode.com>
          Jeremy Rubin <j@rubin.io>
  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0349
  Status: Draft
  Type: Standards Track
  Created: 2024-11-14
  License: BSD-3-Clause
</pre>